### PR TITLE
Fix debug modes

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: js/flightlog_fielddefs.js
Comment (ctzsnooze) [inline js/flightlog_fielddefs.js:547]: Yes this is correct, what I did wasn't for the correct version.

**Review by ctzsnooze — APPROVED** [review]
Looks good.  Tested!

**Pull request merged**
